conf/test_mouse.config

```diff
@@ -17,5 +17,9 @@ params {
 
     // Input data
     input = "https://github.com/nf-core/test-datasets/raw/metapep/samplesheets/input.mouse.csv"
+
+    // Define supported peptide lengths for the mouse allele models
```

Comment:
```suggestion
    // Define peptide lengths that should be used for prediction 
    // (for the provided mouse alleles the SYFPEITHI model only supports peptide length 9)
```
maybe like this?

Reply:
I edited the suggestion, what do you think now? :)